### PR TITLE
Changelog for v2.30.2

### PR DESCRIPTION
**Pull request by nareddyt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Release 2.30.2 01-09-2021
+
+- Update Envoy to top of tree (#588)
```
Comment (TAOXUY) [inline CHANGELOG.md:3]: Could you add the version number or commit date?

Comment (shuoyang2016) [inline CHANGELOG.md:3]: nit: including the sha may be more informative

**Review by shuoyang2016 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (nareddyt) [reply CHANGELOG.md]: Done

Comment (nareddyt) [reply CHANGELOG.md]: Done

**Review by TAOXUY — APPROVED** [review]

**Pull request merged**